docs/concepts/aws.md

```diff
@@ -42,3 +42,35 @@ I0408 05:35:22.865279       1 log.go:172] http: TLS handshake error from 127.0.0
 Those logs are harmless and are caused by AWS ELBs opening TCP connections to `kube-apiserver` to probe for availability, without performing a full TLS handshake. Unfortunately, AWS ELBs do not support TLS for probe requests at the time of writing.
 
 There is ongoing [upstream](https://github.com/kubernetes/kubernetes/pull/91277) work to resolve this issue.
+
+## Using Container Linux Config snippets
```

Comment:
This is repeated in AWS and Packet documents, we should have it in one place and link to it here.

How about adding a `Customization` section in `docs/concepts/flatcar-container-linux.md`?

We can have some introductory text like so:

> Flatcar Container Linux can be customized via Container Linux Configs that are interpreted by [Ignition](https://docs.flatcar-linux.org/ignition/what-is-ignition/).
> 
> Lokomotive supports defining...

And make sure we mention only AWS and Packet platforms support CLC snippets.

Then we can have a `Flatcar Container Linux Customization` section here linking to that.